src/prop.ts

```diff
@@ -245,13 +245,13 @@ function baseProp(
 
     switch (whatis) {
       case WhatIsIt.ARRAY:
+        const virtualSchemaArrayItem = buildSchema(Type, {
```

Comment:
could you combine this buildSchema call with the `case WhatIsIt.NONE` buildSchema call? (and eventually add it into `case WhatIsIt.MAP`)

Reply:
i will try to do this later